demuxer.go

```diff
@@ -78,6 +84,13 @@ func DemuxerOptPacketsParser(p PacketsParser) func(*Demuxer) {
 	}
 }
 
+// DemuxerOptPacketSkipper returns the option to set the packet filter
```

Comment:
Could you update this comment to `// DemuxerOptPacketSkipper returns the option to set the packet skipper`

Reply:
Done